.github/pull_request_template.md

```diff
@@ -0,0 +1,15 @@
+<!--Hi there! If you aren't submitting a blog post, you can delete this message.
+
+Thank you for submitting a blog post! We want to make sure content produced for Quansight Labs is accessible, so please address the the following guidelines in your post.-->
+
+
+Text styling
+- [ ] The blog is written with [plain language](https://www.plainlanguage.gov/guidelines/) (where relevant).
+- [ ] If there are headers, they use the proper header tags in order to do so.
```

Comment:
This has come and bitten a few folks - can we add an additional item or extend this to "There is only one level-one header"